functions.h

```diff
@@ -117,13 +119,20 @@ unique_ptr<T> make_unique(Args&& ...args) {
 namespace navitia {
 
 /**
- * speudo natural sort:
- * if both string cary integer, we compare them, else we compare the string
+ * pseudo natural sort:
+ * if both string carry integer, we compare them, else we compare the string
  */
 struct pseudo_natural_sort {
     bool operator() (const std::string&, const std::string&) const;
 };
 
+/**
+ * strip_accents_and_lower:
+ * We compare strings: integers and accents are taken in account for the sort
```

Comment:
This comment doesn't seem relevant :confused: 
You probably want to update or remove it.

Reply:
Alright, I removed it